inc/user.class.php

```diff
@@ -3346,9 +3347,7 @@ static function getSqlSearchResult ($count = true, $right = "all", $entity_restr
             }
 
             if (count($users)) {
-               $where = " `glpi_users`.`id` IN (".implode(",", $users).")";
-            } else {
-               $where = '0';
```

Comment:
Not sure, but this line was made to have an empty result. Removing it will change the result of this function.

Reply:
I did not see any changes in my tests (mostly on users dropdowns); so I guess this is ok. I do not get the point to get any empty result here.